.github/workflows/versioned-report.yml

```diff
@@ -47,6 +51,16 @@ jobs:
             for impl in $(echo "$IMPLEMENTATIONS" | jq -r '.[]'); do
               if [ -f "implementations/$impl/matrix-versions.json" ]; then
                 MATRIX=$(echo $MATRIX | jq --arg impl "$impl" '. + [$impl]')
+              elif [ ! -d "implementations/$impl" ]; then
```

Comment:
If you have time could you write up an issue ticket which describes the steps we'd need in order to remove this shell code and replace it with another bowtie subcommand? I really want to kill all these ad hoc scripts, they have already grown unmaintainable...

Reply:
We already have #1928 issue to create a command to collect the available versions for the implementation. Would you like to create another issue (linked to the existing one) to replace the shell code with the new command?